src/components/SearchQueryFetch/SolrConnectorDemo/index.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+
+class SolrConnectorDemo extends React.Component {
+  constructor(props) {
+    super(props);
+    this.onSubmit = this.onSubmit.bind(this);
+    this.state = {
+      solrSearchUrl: "http://drupalvm.corporate:8983/solr/d7/select",
+      query: "memory",
+      filter: "",
+      fetchFields: ""
+    }
+  }
+
+  onSubmit(event) {
+  // onLoad(event) {
+    event.preventDefault();
+    let searchParams = {
+      solrSearchUrl: this.state.solrSearchUrl,
+      query: this.state.query,
+      filter: [this.state.filter],
+      fetchFields: this.state.fetchFields.split(" "),
+      offset: 0,
+      limit: 10,
+      facet: {
+        item_bundle: {
+          type: "terms",
+          field: "ss_item_bundle",
+          limit: 20
+        },
+        topics: {
+          type: "terms",
+          field: "ss_node$type",
+          limit: 20
+        },
+      },
+      highlightParams: {
+        "hl": "true",
+        "hl.fl": "content",
+        "hl.snippets": 1,
+        "hl.fragsize": 500
+      }
+    };
+    this.props.doSearch(searchParams);
+  }
+
+  render() {
+    return <div>
+      <form className="inputForm" onSubmit={() =>  this.onSubmit()}>
```

Comment:
This should fix it:

```javascript
onSubmit={(e) => this.onSubmit(e)}
```

Think it's not working because the onSubmit function (defined on line 15) needs the `event` parameter!